From 20edc2b465f0fcf215b47a6651b19ece1b317251 Mon Sep 17 00:00:00 2001
From: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
Date: Wed, 21 Jun 2017 09:57:45 -0700
Subject: [PATCH 0006/1240] thermal: int340x: processor_thermal: Add Coffee
 Lake device id

Add new PCI id for Coffee lake processor thermal device.

Signed-off-by: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
---
 drivers/thermal/int340x_thermal/processor_thermal_device.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/thermal/int340x_thermal/processor_thermal_device.c b/drivers/thermal/int340x_thermal/processor_thermal_device.c
index e724a23..1d9f524 100644
--- a/drivers/thermal/int340x_thermal/processor_thermal_device.c
+++ b/drivers/thermal/int340x_thermal/processor_thermal_device.c
@@ -32,6 +32,7 @@
 
 /* CannonLake thermal reporting device */
 #define PCI_DEVICE_ID_PROC_CNL_THERMAL	0x5a03
+#define PCI_DEVICE_ID_PROC_CFL_THERMAL	0x3E83
 
 /* Braswell thermal reporting device */
 #define PCI_DEVICE_ID_PROC_BSW_THERMAL	0x22DC
-- 
2.7.4

